controller/instance_manager_controller.go

```diff
@@ -1276,7 +1276,7 @@ func (imc *InstanceManagerController) startMonitoring(im *longhorn.InstanceManag
 		done:                   false,
 		monitorVoluntaryStopCh: monitorVoluntaryStopCh,
 		// notify monitor to update the instance map
-		updateNotification: false,
+		updateNotification: true,
```

Comment:
I suspect this was always intended to be true, given the comment.

Reply:
Not sure why this was always intended to be true... When the system is stable, most of the time this should be false.

Reply:
What I mean to say is that it seems like it is always desirable to set this to true when starting the monitor (so we don't have to wait engineapi.MaxPollCount for the first invocation of pollAndUpdateInstanceMap). We usually don't actually wait that long, because some process creation event causes a change in process state and updateNotification flips to true on its own. However, in the situation observed in https://github.com/longhorn/longhorn/issues/5809, this is not the case, delaying stats updates.

I read the comment as meaning we want to set it to true on monitor startup. Certainly I agree that updateNotification should spend most of its life as false. Sorry for the confusion!

Reply:
Thank you for the clarification! Yes, this should be true during monitor startup.